src/schemes.cpp

```diff
@@ -393,14 +393,15 @@ bool PopSchemeMPL::FastAggregateVerify(
     const vector<uint8_t> &message,
     const G2Element &signature)
 {
+    if (pubkeys.size() <= 0) return false;
```

Comment:
Actually the infinity element signature should validate if there are no messages or public keys. I think we forgot to handle this edge case for FastAggregateVerify

Reply:
We don't have to address this is this PR. But the behavior should be the same for all schemes when verifying with no keys/message I think

Reply:
Hm, actually it shouldn't matter in this case if i got it right. You will always have a message here which means https://github.com/Chia-Network/bls-signatures/blob/5d5da2778916cf39e5908372a36fd20c14f88724/src/schemes.cpp#L39 will anyway return `BAD` aka `false` and it couldn't even run into the "allow infinity" case.

Reply:
Oh okay that makes sense. These methods take in one public key and one message so there is no need to check